src/main/kotlin/com/atlassian/performance/tools/report/result/PrintableInteractionStats.kt

```diff
@@ -0,0 +1,24 @@
+package com.atlassian.performance.tools.report.result
+
+import com.atlassian.performance.tools.report.MeanAggregator
+import com.atlassian.performance.tools.report.api.result.InteractionStats
+
+internal class PrintableInteractionStats(
+    stats: InteractionStats,
+    actions: List<String>
+) {
+    internal val cohort: String = stats.cohort
```

Comment:
When the class is `internal`, the members will effectively be `internal` or even less visible